### PR TITLE
First pass at creating config files to deploy the wiki app to fly.io

### PR DESCRIPTION
**Pull request by neelav**

### CODE REVIEW

File: docker/wiki.fly.nginx.conf [hunk 1]
```diff
@@ -0,0 +1,14 @@
+server {
+    listen 80 default_server;
+    root /var/www/public;
+
+    index index.php index.html index.htm;
+
+    try_files $uri /index.php?$args;
+
+    location ~ \.php$ {
+        fastcgi_pass 127.0.0.1:9000;
```
Comment (neelav) [inline docker/wiki.fly.nginx.conf:10]: i copied the file you created and changed this one line.  In the fly.io dockerfile below I combine the php server and nginx server into one docker image.  This simplifies the deployment architecture and keeps me in the free tier.

We should probably use this version of the code for docker compose too

File: docker/wikisrv.fly.dockerfile [hunk 2]
```diff
@@ -0,0 +1,29 @@
+### Setup PHP server
```
Comment (neelav) [inline docker/wikisrv.fly.dockerfile:1]: combines your php and nginx dockerfiles into one file

File: wikisrv.fly.toml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# fly.toml app configuration file generated for wiki-srv on 2023-12-13T14:28:57-08:00
+#
+# See https://fly.io/docs/reference/configuration/ for information about how to use this file.
+#
+
+app = "wiki-srv"
```
Comment (neelav) [inline wikisrv.fly.toml:6]: in total I'll create:

* wiki-srv
* todo-srv
* mysql <--- will create a single db that is shared so I stay in the free tier

File: docker/wiki.fly.nginx.conf (hunk 1 above)
Comment (aaronpk) [reply docker/wiki.fly.nginx.conf:10]: oh cool I didn't know you could do that, I thought I read somewhere that there could be only one server per docker container. yeah let's change this for the main one too.

Comment (neelav) [reply docker/wiki.fly.nginx.conf:10]: it's more idiomatic the way you did it, but then harder to manage since not all tools play nice with docker-compose

**Pull request merged**